src/ensembl/src/content/app/browser/track-panel/track-panel-modal/modal-views/TrackPanelBookmarks.tsx

```diff
@@ -47,7 +47,10 @@ export const ExampleLinks = (props: ExampleLinksProps) => {
         });
 
         return (
-          <div key={`exampleLinks__${exampleObject.object_id}`} className={styles.linkHolder}>
+          <div
+            key={`exampleLinks__${exampleObject.object_id}`}
```

Comment:
I know it’s inherited from previous commit, but I strongly suspect that the key can be simply `exampleObject.object_id`. It just needs to be unique; adding the `exampleLinks` string is not doing anything for that.

Same with `viewedLinks__${previouslyViewedObject.object_id}` below

Reply:
I think I was the culprit in adding these prefixes to the key names to fix an issue that occurs in the `TrackPanelBookmarks` tests randomly. I tried to fixing them on the test side of things itself, and the issues goes away. I'll talk to @jyothishnt about this and get those fixes into this PR. As for the key names we are using in the component itself, what do you suggest we use instead @azangru?

Reply:
> As for the key names we are using in the component itself

Just the `exampleObject.object_id`?

By the way, I have noticed a number of tests complain (although still passing) about non-unique keys in some arrays of components. I suspect this complaint arises from the fact that we are using `faker.lorem.word` to generate random strings, and faker repeats some words. Perhaps a better idea might be to use `faker.random.uuid` (even `faker.random.word` seems to be more random than `faker.lorem.word`)?

Reply:
Yes, that is one reason why this keeps happening, though I wouldn't rely only on `faker.random.word` to generate a unique ID. `faker.random.uuid` will be much better.

Reply:
changed 👍